tutorials/simulation_exporting.ipynb

```diff
@@ -0,0 +1,324 @@
+{
```

Comment:
<p>Suggested change (new text in bold, deletions with strikethrough):</p><p> </p><p>We now extend the <strong>previous example</strong> to the mixed-dimensional case. We can use <code>Exporting</code> as above. In the parameters, we adjust <strong>the</strong> file name to avoid overwriting the previous files. If you inspect the suffixes of the files created, you can see how the exporter deals with multiple time steps by default.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/708/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>